packages/core/src/plugins/splitChunks.ts

```diff
@@ -275,8 +275,8 @@ export const pluginSplitChunks = (): RsbuildPlugin => ({
           chunkSplit.strategy || 'split-by-experience'
         ]({
           defaultConfig,
-          override: override || {},
           forceSplittingGroups,
+          override: override || {},
```

Comment:
Is this an effective change? Changing the order of the object keys does not seem to cause any change.